src/common/tree_renderer.cpp

```diff
@@ -498,13 +498,20 @@ string TreeRenderer::ExtractExpressionsRecursive(ExpressionInfo &state) {
 }
 
 unique_ptr<RenderTreeNode> TreeRenderer::CreateNode(const QueryProfiler::TreeNode &op) {
```

Comment:
My editor is showing this function as `unused`. Can we potentially remove it altogether if it is a relic?

```cpp
	unique_ptr<RenderTreeNode> CreateNode(const QueryProfiler::TreeNode &op);
	unique_ptr<RenderTreeNode> CreateNode(const PipelineRenderNode &op);
```

Reply:
It doesn't compile without them...

Reply:
Then we keep them haha.